mgmtfn/mesosplugin/contiv-isolator/netpluginhandler_test.go

```diff
@@ -0,0 +1,42 @@
+/***
+Copyright 2016 Cisco Systems Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+/*
+import (
+	"testing"
+)
+
+func TestNetpluginHandler(t *testing.T) {
```

Comment:
Can we please create some unit-test for the netplugin handler in contiv-isolator?
Just to verify its making right HTTP calls to netplugin..